radicle-node/src/service.rs

```diff
@@ -868,15 +891,16 @@ where
                     .insert(message.rid, *relayer, message.timestamp)
                 {
                     if updated {
-                        info!(target: "service", "Routing table updated for {} with seed {relayer}", message.rid);
+                        info!(target: "service", "handle_announcement: Routing table updated for {} with seed {relayer}", message.rid);
                     }
                 }
                 // TODO: Buffer/throttle fetches.
-                if self
+                let repo_policy = self
                     .tracking
-                    .is_repo_tracked(&message.rid)
-                    .expect("Service::handle_announcement: error accessing tracking configuration")
-                {
+                    .repo_policy(&message.rid)
+                    .expect("handle_announcement: error accessing repo tracking configuration");
```

Comment:
```suggestion
                    .expect("Service::handle_announcement: error accessing repo tracking configuration");
```

Reply:
I can make the change. Just to clarify, the module name `service` appears in the log line already.